test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,22 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        assertNoNull();
+        assertNoNull(1, 2, 3);
+        assertNoNull("a", "b", "c");
+
+        assertHasNull(null, 2, 3);
+        assertHasNull("a", null, null);
```

Comment:
Well organised and efficient test cases, can try to include newly instantiate object such as `new Object()` too